auth/jwt.go

```diff
@@ -111,7 +111,8 @@ func (id *jwtIdentity) Valid() error {
 	}
 
 	now := jwt.TimeFunc().UTC().Unix()
-	if now < id.IssuedAt {
+	// provide tolerance for clockdrifting slower
+	if now < id.IssuedAt-int64(ClockDriftDelta.Seconds()) {
```

Comment:
2. This is a check of the token at the client side on the validity of the token it has. This particular part is to check if the token has started to become valid (not expired) So, using clockdriftdelta should be correct.
3. The code below should check the expiration with expire.. I agree that we probably need to add the possible timedriftdelta to this as well.